.github/workflows/ci.yml

```diff
@@ -55,7 +55,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - uses: haskell/actions/setup@v1
+    - uses: haskell/actions/setup@v1.2
```

Comment:
I realized that specifying two components for this action provides the perfect balance between stability and convenience. `v1.3` might have breaking changes so we want to update it explicitly with smth like @dependabot. But it's becoming tedious to update all our packages with each new minor version. So, just a practical trade-off.